torax/data/third_party/geo/README

```diff
@@ -27,4 +27,11 @@ The mapping of datafile: LICENSE for the geometry files in this directory are as
 All associated LICENSE files are also located in this directory.
 
 ITER_hybrid_citrin_equil_cheasedata.mat2cols: LICENSE_PINT
-eqdsk_cocos02.eqdsk: LICENSE_PINT
\ No newline at end of file
+eqdsk_cocos02.eqdsk: LICENSE_PINT
+
+The IMAS netCDF file ITERhybrid_rampup_11_time_slices_COCOS17_IDS_ddv4.nc was
+generated with CHEASE using p' and ff' from a TORAX simulation using
+iterhybrid_rampup.py example config. It contains equuilibrium for each of the
+time slices of the TORAX simulation. The plasma boundaries outline R and Z
```

Comment:
"The plasma boundaries outline R and Z were prescribed constant for all time slices"

this also doesn't quite read correctly to me, could be my lack of physics knowledge though!